cp3pt0-deployment/setup_singleton.sh

```diff
@@ -16,6 +16,8 @@ CHANNEL="v4.0"
 SOURCE="opencloud-operators"
```

Comment:
@qpdpQ 

I think we could remove `SOURCE` from this script. This was totally replaced by separated `CERT_MANAGER_SOURCE` and `LICENSING_SOURCE`